docs/readme.md

```diff
@@ -1 +1,124 @@
-readme.file
+# Chicken & Fries
+
+This is a project for the Web Application Integration online class at Full Sail University
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine.
+
+### Prerequisites
+
+To run this project on you local computer you need to have NodeJs and NPM installed on your machine
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+First clone or download the files on your machine
+
+Place the files in an easy to access folder on your machine
+
+```
+Desktop
+```
+Open your terminal window and run the following command
+```
+nodemon bin/www
+```
+Open your favorite browser and go to
+```
+localhost:3000
+```
+
+#Table Of Contents
+* [Brief]()
+    * [Audience]()
+    * [Problem]()
+    * [Solution]()
+    * [Unique Value Proposition]()
+    * [Pitch]()
+    
+* [Spec]()
+    * [Feature Definitions]()
+    * [Sitemap]()
+    * [Interface]()
+        * [Information Architecture]()
+        * [Browser Support]()
+        
+    * [Infrastructure]()
+        * [Technical Requirements]()
+        * [Programming Languages]()
+        * [Integrations]()
+        * [Deployment Workflow]()
+        * [Web Host]()
+* [Project Milestones]()
+    * [Initial Pitch]()
+    * [Initial Project Spec]()
+    * [MVP]()
+    * [Pivot]()
+    * [Alpha Release]()
+    * [Beta Release]()
+    * [Public release]()
+
+#Brief
```

Comment:
Currently some of this content is duplicated or split between this main readme file and more specific sub-files. You have two ways you can approach this:
1. Keep all the specification information in this file like it is now (with requested changes) and then remove the extra files (brief.md, milestones.md, etc).
2. Use this readme file as just a table of contents, with links to those extra files.

Reply:
fixed it